bond_test.go

```diff
@@ -51,10 +63,23 @@ type User struct {
 	Username  string `db:"username"`
 }
 
+func (u User) AfterCreate(tx bond.Session) {
```

Comment:
however, I would call it.. `AfterCreate(sess bond.Session)`  ... since the session can be a normal conn or a tx .. its up to the previous caller context